woocommerce/woocommerce-ios
# Swift linting

Closes #731. All styles come from _[Swift Style, Second Edition: An Opinionated Guide to an Opinionated Language](https://pragprog.com/book/esswift2/swift-style-second-edition)_. If you have a style preference not found here, please comment below!

## To test
- check out this branch
- in the command line, `cd` into the repo
- run the `swiftlint` command

There will be 24 `line_length` violations. Every one of them are string localizations, and I'm not sure we _want_ to create concatenated multi-line comments for the translators.

Sorry about the size of the PR. Didn't want to break up styling into multiple reviews.

Update release notes:
